get-started/the-stack.md

```diff
@@ -7,97 +7,147 @@ mapped_pages:
+### {{kib}} [stack-components-kibana]
 
-#### {{ls}} [stack-components-logstash]
+With {{kib}}, you can:
 
-{{ls}} is a data collection engine with real-time pipelining capabilities. It can dynamically unify data from disparate sources and normalize the data into destinations of your choice. {{ls}} supports a broad array of input, filter, and output plugins, with many native codecs further simplifying the ingestion process. [Learn more about {{ls}}](logstash://reference/index.md).
+- Use **Discover** to interactively search and filter your raw data.
+- Build custom visualizations like charts, graphs, and metrics with tools like **Lens**, which offers a drag-and-drop experience.  
+- Assemble your visualizations into interactive dashboards to get a comprehensive overview of your information.
+- Analyze geospatial data using the powerful **Maps** application.
+- Manage resources such as processors, pipelines, data streams, trained models, and more.
 
+Each solution or project type provides access to customized features in {{kib}} such as built-in dashboards and [](/explore-analyze/ai-assistant.md).
 
-### Store [_store]
+{{kib}} also has [query tools](/explore-analyze/query-filter/tools.md) such as **Console**, which provides an interactive way to send requests directly to the {{es}} API and view the responses.
+For secure, automated access, you can create and manage API keys to authenticate your scripts and applications.
 
-#### {{es}} [stack-components-elasticsearch]
+[Learn more about {{kib}}](/explore-analyze/index.md).
 
-{{es}} is the distributed search and analytics engine at the heart of the {{stack}}. It provides near real-time search and analytics for all types of data. Whether you have structured or unstructured text, numerical data, or geospatial data, {{es}} can efficiently store and index it in a way that supports fast searches. {{es}} provides a REST API that enables you to store data in {{es}} and retrieve it. The REST API also provides access to {{es}}'s search and analytics capabilities. [Learn more about {{es}}](/get-started/index.md).
+## Ingest [_ingest]
 
+There are multiple methods for ingesting data.
+The best approach depends on the kind of data you're ingesting and your specific use case.
+For example, you can collect and ship logs, metrics, and other types of data with {{agent}} or collect detailed performance information with {{product.apm}}.
+If you want to transform and enrich data before it's stored, you can use {{es}} ingest pipelines or {{ls}}.
 
-### Consume [_consume]
+Trying to decide which ingest components to use? Refer to [](/manage-data/ingest.md) and [](/manage-data/ingest/tools.md).
 
-Use {{kib}} to query and visualize the data that’s stored in {{es}}. Or, use the {{es}} clients to access data in {{es}} directly from common programming languages.
+### {{agent}} and {{integrations}}[stack-components-agent]
```

Comment:
maybe no capital I here for sentence case reasons? integrations docs use lowercase `integrations` in sentences